cfme/tests/cloud/test_tenant.py

```diff
@@ -13,7 +13,8 @@
 
 @pytest.yield_fixture(scope='function')
 def tenant(provider, setup_provider):
-    tenant = Tenant(name=fauxfactory.gen_alphanumeric(8), provider=provider)
+    collection = TenantCollection()
```

Comment:
**REQUIRED**: We should always pass an appliance object to a collection, this is fundamental to the migration to collections. The fixture can just add appliance on line 15 and it will get it via a global fixture.